### PR TITLE
Fix email queue resend process

### PR DESCRIPTION
**Pull request by patrykgruszka**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
I've noticed that Mautic never attempts to resend email through queue and sets DNC flag after first try (part of the code was unreachable). This PR resolves this issue.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Enable email queue
3. Send failing email (e.g. to server that does not respond)
4. Email queue command should attempt to resend email few times before removing it and setting DNC flag on contact

### CODE REVIEW

File: app/bundles/EmailBundle/EventListener/EmailSubscriber.php [hunk 1]
```diff
@@ -24,29 +24,14 @@
 
 class EmailSubscriber implements EventSubscriberInterface
 {
-    /**
-     * @var AuditLogModel
```
Comment (escopecz) [inline app/bundles/EmailBundle/EventListener/EmailSubscriber.php]: Don't get me wrong. I hate useless comments. But these property annotations are valuable for some IDEs. Could you please add them back?

File: app/bundles/EmailBundle/EventListener/EmailSubscriber.php [hunk 2]
```diff
@@ -145,7 +130,7 @@ public function onEmailResend(Events\QueueEmailEvent $event)
                 $stat->upRetryCount();
 
                 $retries = $stat->getRetryCount();
-                if (true || $retries > 3) {
```
Comment (escopecz) [inline app/bundles/EmailBundle/EventListener/EmailSubscriber.php:148]: Wow, this is awkward. Good catch!

**Review by escopecz — CHANGES_REQUESTED** [review]

File: app/bundles/EmailBundle/EventListener/EmailSubscriber.php (hunk 1 above)
Comment (patrykgruszka) [reply app/bundles/EmailBundle/EventListener/EmailSubscriber.php]: Sure, sorry for that. I think just my IDE doesn't like them. Reverted.

**Review by escopecz — DISMISSED** [review]

File: app/bundles/EmailBundle/EventListener/EmailSubscriber.php (hunk 1 above)
Comment (escopecz) [reply app/bundles/EmailBundle/EventListener/EmailSubscriber.php]: Thanks!

**Review by kuzmany — APPROVED** [review]
Looks good. Thanks 👍

**Pull request merged**